packages/yourloops/components/dashboard-widgets/medical-files/medical-record-list.tsx

```diff
@@ -152,41 +149,45 @@ const MedicalRecordList: FunctionComponent<CategoryProps> = (props) => {
       </Typography>
       {medicalRecords
         ? <List className={classes.list}>
-          {medicalRecords.map((medicalRecord, index) => (
-            <ListItem
-              dense
-              divider
-              key={index}
-              aria-label={`record-${medicalRecord.id}`}
-              className={`${classes.hoveredItem} ${medicalRecord.id === hoveredItem ? 'selected' : ''}`}
-              onClick={() => { onClickMedicalRecord(medicalRecord) }}
-              onMouseOver={() => { setHoveredItem(medicalRecord.id) }}
-              onMouseOut={() => { setHoveredItem(undefined) }}
-            >
-              <ListItemIcon>
-                <DescriptionOutlinedIcon />
-              </ListItemIcon>
-              <ListItemText>
-                {t('medical-record-pdf', { pdfName: buildFileName(medicalRecord.creationDate, index) })}
-              </ListItemText>
-              {user.isUserHcp() && medicalRecord.id === hoveredItem &&
-                <ListItemSecondaryAction>
-                  <Tooltip title={t('delete')}>
-                    <IconButton
-                      edge="end"
-                      size="small"
-                      disableRipple
-                      disableFocusRipple
-                      aria-label={t('delete')}
-                      onClick={() => { onDeleteMedicalRecord(medicalRecord) }}
-                    >
-                      <TrashCanOutlined />
-                    </IconButton>
-                  </Tooltip>
-                </ListItemSecondaryAction>
-              }
-            </ListItem>
-          ))}
+          {medicalRecords.map((medicalRecord, index) => {
+            const medicalRecordName = buildFileName(medicalRecord.creationDate, index)
+            return (
+              <ListItem
+                key={index}
+                dense
+                divider
+                className={classes.listItem}
+              >
+                <ListItemButton
+                  onClick={() => {
+                    onClickMedicalRecord(medicalRecord, medicalRecordName)
+                  }}
+                >
+                  <ListItemIcon>
+                    <DescriptionOutlinedIcon />
+                  </ListItemIcon>
+                  <ListItemText>
+                    {t('medical-record-pdf', { pdfName: medicalRecordName })}
+                  </ListItemText>
+                </ListItemButton>
+                <Tooltip title={t('delete-medical-record', { date: medicalRecordName })}>
+                  <IconButton
+                    data-testid="delete-medical-record"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏This one is not tested in integration. Could be nice to have a test for it, it will also secure that a patient could not delete at all a medical record ;)